packages/contentful--app-scripts/lib/utils.js

```diff
@@ -81,7 +98,33 @@ function getActionsManifest() {
   ----------------------------`);
     console.log('');
 
-    return manifest.actions.map((action) => ({ parameters: [], ...action })); // adding required parameters
+    const actions = manifest.actions.map((action) => {
+      const allowedNetworks = Array.isArray(action.allowedNetworks)
+        ? [...action.allowedNetworks, 'api.contentful.com', 'upload.contentful.com', '127.0.0.1']
+        : ['api.contentful.com', 'upload.contentful.com', '127.0.0.1'];
+
+      const validAllowedNetworks = allowedNetworks
```

Comment:
You can simplify this with `hasInvalidNetwork = allowedNetworks.map().some(netWork => !isValid())`